src/main/java/com/socrata/datasync/job/IntegrationJob.java

```diff
@@ -114,13 +136,19 @@ public IntegrationJob(String pathToFile, boolean ignoreControlInconsistencies) t
         // first try reading the 'current' format
         ObjectMapper mapper = new ObjectMapper();
         IntegrationJob loadedJob;
+
+        boolean oldHack = jacksonHack.get();
         try {
+            jacksonHack.set(false);
             loadedJob = mapper.readValue(new File(pathToFile), IntegrationJob.class);
         } catch (IOException e) {
             // if reading new format fails...try reading old format into this object
             loadOldSijFile(pathToFile);
```

Comment:
Is this the `Then, when  it's saved, it'll be set true over in the IntegrationJobTab, and the resolution will be frozen.` part? I don't really understand--in both cases, won't this always be set True in the finally statement?